app/backend/server/auth.py

```diff
@@ -0,0 +1,157 @@
+import os
```

Comment:
how connection with google/facebook is done?

Reply:
it will be added in another issue.
currently it is not a must have for the beta